uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/pom.xml

```diff
@@ -181,8 +192,9 @@
           <useSystemClassLoader>true</useSystemClassLoader>
           <useManifestOnlyJar>true</useManifestOnlyJar>
           <forkMode>once</forkMode>
+          <argLine>-Djdk.attach.allowAttachSelf=true</argLine>
           <!--<parallel>false</parallel>-->
-          <!-- ensure we don't inherit a byteman jar from any env settings -->
+          <!-- ensure we don't inherit a byteman jar from any env senattings -->
```

Comment:
```suggestion
          <!-- ensure we don't inherit a byteman jar from any env settings -->
```

Reply:
Fixed in https://github.com/kiegroup/appformer/pull/1059/commits/7643c02745b674bd5a76e230220e2d9f457d406f.